### ISSUE TITLE
Ability to select from alternate titles (& Different UI for them)

### ISSUE THREAD

**Issue by JDKamalakar** [mihonapp/mihon#615]
### Describe your suggested feature

I am thinking of something like neko (Tachi Fork) offers. 
Show alternate in similar UI to tags & ability to replace anyone of them with the title. 
![Screenshot_20240402-124118](https://github.com/mihonapp/mihon/assets/101495154/7be0d03d-88b1-4859-933b-1d3e7aa769b3)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.16.4](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by BrutuZ** [comment]
Seems like basically a duplicate of #5 which would get fixed by #398

**Comment by JDKamalakar** [comment]
Ok It is same as them.

**Comment by sirlag** [comment]
I don't think it is a duplicate. Allowing sources to provide alternative titles removes the manual work of editing manga info per title. 
The replacement option *does* get fixed by edit manga info, but it feels like there is room for an improvement.

I do know extension API changes are all currently on hold though, so I wouldn't expect it as a feature any time soon.
